src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState: [],
```

Comment:
- [ ] Nice work so far in creating your initialState, but as per the project you are to have an empty book array
 
```
initialState: {
    books: [],
  },
```